pints/tests/test_residuals_diagnostics.py

```diff
@@ -0,0 +1,199 @@
+#
+# Test the methods in pints.residuals_diagnostics
+#
+# This file is part of PINTS
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing informating, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+import pints
```

Comment:
We tend to do imports alphabetically, but I'm not sure we've always been consistent

Reply:
(Except `from .. import ...`, those always go below the `import ...`)